executor/src/main/java/org/k8loud/executor/action/kubernetes/KubernetesAction.java

```diff
@@ -2,19 +2,16 @@
 
 import data.Params;
 import io.fabric8.kubernetes.client.KubernetesClient;
-import io.fabric8.kubernetes.client.KubernetesClientBuilder;
 import org.k8loud.executor.action.Action;
 import org.k8loud.executor.exception.ActionException;
 
-import java.util.Map;
-
 public abstract class KubernetesAction extends Action {
     protected KubernetesClient client;
 
     protected KubernetesAction(Params params) throws ActionException {
         super(params);
         //TODO how to configure??
-        this.client = new KubernetesClientBuilder().build();
```

Comment:
why?

Reply:
Firstly was not working probably because of my changes. Right now it's good, reverted